src/CTA.Rules.Actions/ClassActions.cs

```diff
@@ -259,5 +260,210 @@ ClassDeclarationSyntax AppendConstructorExpression(SyntaxGenerator syntaxGenerat
             }
             return AppendConstructorExpression;
         }
+
+        public Func<SyntaxGenerator, ClassDeclarationSyntax, ClassDeclarationSyntax> GetCreateConstructorAction(string types, string identifiers)
```

Comment:
@jonlouie @Eruanion @marknfawaz please review this class... leaving to you folks

Reply:
@all it's good idea to give some description on methods what it's doing if you think this would help on boarding new members